lib/players/computer.rb

```diff
@@ -0,0 +1,122 @@
+module Players
+
+  class Computer < Player
+
+    attr_accessor :board
+
+    CORNER_CELLS = [0, 2, 6, 8]
+    CROSS_CELLS = [1, 3, 5, 7]
+
+    def winning_move
+      three_in_a_row = Game::WIN_COMBINATIONS.find do |win_combination|
+        (board.cells[win_combination[0]] == token &&
+        board.cells[win_combination[1]] == token &&
+        board.cells[win_combination[2]] == " ") ||
+        (board.cells[win_combination[0]] == token &&
+        board.cells[win_combination[1]] == " " &&
+        board.cells[win_combination[2]] == token) ||
+        (board.cells[win_combination[0]] == " " &&
+        board.cells[win_combination[1]] == token &&
+        board.cells[win_combination[2]] == token)
+      end
+      winning_cell = three_in_a_row.find { |cell| board.cells[cell] == " " } if three_in_a_row != nil
```

Comment:
You do not need to compare to nil -- rely on the implicit return value of the expression. For example:

```
winning_cell = three_in_a_row.find { |cell| board.cells[cell] == " " } if three_in_a_row
```